ReactiveCocoa/Swift/Property.swift

```diff
@@ -580,14 +580,14 @@ public final class MutableProperty<Value>: MutablePropertyProtocol {
 	/// - parameters:
 	///   - initialValue: Starting value for the mutable property.
 	public init(_ initialValue: Value) {
+		(signal, observer) = Signal.pipe()
+
 		/// Need a recursive lock around `value` to allow recursive access to
 		/// `value`. Note that recursive sets will still deadlock because the
 		/// underlying producer prevents sending recursive events.
-		let lock = RecursiveLock()
-		lock.name = "org.reactivecocoa.ReactiveCocoa.MutableProperty"
-		_atomic = Atomic(initialValue, mutex: lock)
-
-		(signal, observer) = Signal.pipe()
+		_atomic = RecursiveAtomic(initialValue,
+															name: "org.reactivecocoa.ReactiveCocoa.MutableProperty",
+															didSet: observer.sendNext)
```

Comment:
[Indentation here is wrong](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/dc104c86b59533a5de2c75813d65bf96b24c516d/ReactiveCocoa/Swift/Property.swift#L589).